microshift_networking/microshift-network-policy/microshift-creating-network-policy.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-creating-network-policy"]
+= Creating network policies
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/common-attributes.adoc[]
+:context: microshift-creating-network-policy
+
+toc::[]
+
+You can create a network policy for a namespace.
+
+[NOTE]
+====
+Firewalld rules run before any `NetworkPolicy` is enforced.
```

Comment:
It'd be better to move this line to the `nw-networkpolicy-allow-external-clients` section. Since firewalld cannot control the cluster's internal traffic.

Reply:
Oh, right. Thanks!